src/Utils/StyleAttributesUtils.php

```diff
@@ -327,6 +327,44 @@ public static function get_align_class_and_style( $attributes ) {
 		return null;
 	}
 
+	/**
+	 * Get class and style for text align from attributes.
+	 *
+	 * @param array $attributes Block attributes.
+	 *
+	 * @return (array | null)
+	 */
+	public static function get_text_align_class_and_style( $attributes ) {
+
+		$text_align_attribute = isset( $attributes['textAlign'] ) ? $attributes['textAlign'] : null;
```

Comment:
This is a good suggestion. I have double-checked and it seems that it's safe to use since WordPress now requires PHP 7.0.

That said, I have just followed the function above, and tried to keep the code consistent. I don't have a strong preference, to be honest :)

https://github.com/woocommerce/woocommerce-blocks/blob/e79766ab116827f4744d4b940a09e1e9892ebacf/src/Utils/StyleAttributesUtils.php#L286

Reply:
> I have double-checked and it seems that it's safe to use since WordPress now requires PHP 7.0.

Yes it is ;)